babel.config.js

```diff
@@ -1,24 +1,25 @@
-// module.exports = {
-//   presets: [
-//     [
-//       'espressive',
-//       {
-//         targets: {
-//           node: 'current',
-//         },
-//       },
-//     ],
-//   ],
-// };
-
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-react'],
-  plugins: [
+  presets: [
     [
-      '@babel/plugin-proposal-class-properties',
+      '@babel/preset-env',
       {
+        modules: false,
         loose: true,
       },
     ],
+    '@babel/preset-react',
+  ],
+  plugins: [
+    '@babel/plugin-proposal-optional-chaining',
```

Comment:
@Espressive/front-end This is something that would be great to have in all of our applications to be honest. https://github.com/tc39/proposal-optional-chaining